.github/workflows/main.yaml

```diff
@@ -24,4 +24,4 @@ jobs:
       - name: "Run Pytest"
         run: pytest
       - name: "Run PyLint"
-        run: pylint --fail-under=1 *.py
+        run: pylint --fail-under=1 $(find . -name "*.py")
```

Comment:
I would like some clarification on what this change will do to the job.

Reply:
This finds python files instead of *.py file